src/hacbs/types/pipelineRun.ts

```diff
@@ -142,7 +142,7 @@ export type PipelineRunStatus = {
 
 export type PipelineRunKind = K8sResourceCommon & {
   spec: {
-    pipelineRef?: { name: string; bundle?: string };
+    pipelineRef?: { name: string };
```

Comment:
Please revert this change as `Tekton` now supports [bundle](https://github.com/tektoncd/pipeline/blob/main/docs/pipelineruns.md#tekton-bundles) field to support remotely defined pipelines.
```suggestion
    pipelineRef?: { name: string, bundle?: string };
```

Reply:
done, thanks!